docs/source/1.0/guides/model-linters.rst

```diff
@@ -128,6 +128,68 @@ Example:
     ]
 
 
+.. _NoninclusiveTerms:
+
+NoninclusiveTerms
+=================
+
+Validates that all text content in a model (i.e. shape names, member names,
+documentation, trait values, etc.) do not contain words that perpetuate cultural
+biases. This validator has a built-in set of bias terms that are commonly found
+in APIs along with suggested alternatives.
+
+Noninclusive terms are case-insensitively substring matched and can have any
+number of leading or trailing whitespace or non-whitespace characters.
+
+Rationale
+    Intent doesn't always match impact. The use of noninclusive language like
+    "whitelist" and "blacklist" perpetuates bias through past association of
+    acceptance and denial based on skin color. Other words should be used that
+    are not only inclusive, but more clearly communicate meaning. Words like
+    allowList and denyList much more clearly indicate that something is
+    allowed or denied.
+
+Default severity
+    ``WARNING``
+
+Configuration
+    .. list-table::
+       :header-rows: 1
+       :widths: 20 20 60
+
+       * - Property
```

Comment:
Will add documentation. It is worth noting there is a non-trivial, maybe counter intuitive defaults behavior:

Though `appendDefaults` defaults to false, if `noninclusiveTerms` mappings is entirely unset or empty, `appendDefaults` behaves as if it were true -- the built in mappings are present. `noninclusiveTerms` has to be non-empty before `appendDefaults` behavior applies. If this behavior is acceptable, then I'll focus on clear and concise documentation for it. If not, then I should change the implementation.

Reply:
Property structure have changed a bit. But current properties are documented for required or not, along with what the default values are

Reply:
Is the reasoning for this structure change because you weren't happy with the behavior? I feel like a terms map and a defaults boolean that defaults to true would be pretty clear and usable for customers.